core/cli/cli.js

```diff
@@ -118,3 +155,48 @@ function green_checkmark() {
     const chalk = require('chalk');
     return chalk.green('\u2714');
 }
+
+function createScaffold(projectName) {
+    const {homeViewTemplate, homePageTemplate} = require('./templates/homeTemplate');
+    const viewTemplate = homeViewTemplate();
+    const pageTemplate = homePageTemplate(projectName);
+    let currentDir = path.normalize(path.resolve(process.cwd(), projectName));
```

Comment:
I guess we don't need `path.normalize` here. Correct me if I'm wrong.

Reply:
You are probably correct.  I guess I was thinking that if someone wanted to pass an actual path as an argument, we may want to normalize it, but that's probably not something we need to support.  I'll remove it!